packages/fast-components-react-base/src/hypertext/index.ts

```diff
@@ -0,0 +1,4 @@
+import Hypertext from "./hypertext";
+
+export default Hypertext;
+export * from "./hypertext";
```

Comment:
I think we need a newline here

Reply:
Are we saying that we should have new lines before export statements?  If so, should this be a new rule on tslint package?

Reply:
I sort of liked grouping by component name.

Reply:
There should be a .ts check for newline at the end of the file. I'm trying to hook up ordering alphabetically in tslint for these import/exports.